app/styles/main.scss

```diff
@@ -23,6 +23,17 @@ html, body {
   height: 100%;
 }
 
+.flexbox-container {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  min-height: 100vh;
+}
+
+.flexbox-spacer {
+  flex-grow: 2;
```

Comment:
So yeah, looks like `flex: auto` has the same result as `flex-grow: 2` in this case